mlflow/sagemaker/cli.py

```diff
@@ -19,11 +19,11 @@ def commands():
 @commands.command("deploy")
 @click.option("--app-name", "-a", help="Application name.", required=True)
 @cli_args.MODEL_PATH
-@click.option("--execution-role-arn", "-e", help="SageMaker execution role", required=True)
-@click.option("--bucket", "-b", help="S3 bucket to store model artifacts", required=True)
+@click.option("--execution-role-arn", "-e", default=None, help="SageMaker execution role")
+@click.option("--bucket", "-b", default=None, help="S3 bucket to store model artifacts")
 @cli_args.RUN_ID
-@click.option("--container", "-c", default="mlflow_sage", help="container name")
-@click.option("--region-name", default="us-west-2", help="region name")
+@click.option("--container", "-c", default=None, help="container name")
```

Comment:
this is a breaking-ish change because the default behavior has changed: 
* previously if a user did not specify this parameter, they'd be using the docker image they uploaded using mlflow (which lives in the user's account's ECR). now if a user does not specify this parameter, they'll use the publicly-available hosted one. 
* now if you want to use a custom image, or an image in your own account, you need to specify the name of the image, which was hidden away before. 

@tomasatdatabricks, @mateiz - do you think it's okay to change this behavior given we are still early? i'm not sure of a great alternative here since I think the default *should* be that you use the prebuilt image instead of having to upload your own.

Reply:
(btw, in @dbczumar's earlier version, he suggested changing the parameter `image` to `image_url` to support using images in other accounts - e.g. SageMaker has published some public images. if we are okay making breaking changes, we should consider that as well.)

Reply:
+1 to  image_url. Also, in retrospect image would be a better name than container.